nixos/modules/virtualisation/containers.nix

```diff
@@ -318,6 +319,16 @@ let
       '';
     };
 
+    hostPort = mkOption {
+      type = types.nullOr types.string;
```

Comment:
Okay, then I think the `example` and description attributes are clearly misleading, because `Allow port forwarding...` sounds like if it is a boolean option. I'd flesh this out as a submodule to avoid future need of deprecation once we want to remove the stringly typed one.

Also, doesn't this make more sense to be a list? What if you want to forward several ports (which I think is not uncommon)?